code-challenges.test.js

```diff
@@ -59,9 +95,30 @@ var nums2 = [109, 5, 9, -59, 8, 24]
 var testArray1 = [3, 7, 10, 5, 4, 3, 3]
 var testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
-
-
+//make test function
+describe("noDupe", () =>{
+    //give test parameters
+    test("returns one array with no duplicate values", () =>{
+        expect(noDupe(testArray1, testArray2)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+    })
+})
 
 
 
 // b) Create the function that makes the test pass.
+//create function for removing values
+const noDupe = (array1, array2) => {
+    //add arrays make empty array for new
+    let combined = [array1 += array2]
+    let newArr =  []
+    //loop over array checking each value
+    for(let i =0; i <combined.length; i++){
+        if(newArr.indexOf(combined[i] === -1)){
+            //push values to new array
+            newArr.push(combined[i])
+        }
+    }
+    return newArr
+}
+
+//it kept returning the whole combined two arrays, I need to figure out how to remove duplicates
```

Comment:
Solid note! I think using a for loop and not methods like sort throws people off with ever more complicated logic in stead of logical steps.  The Idea of an accumulator here is quite helpful. a sorted array that holds a prev value can compare a value that it's iterating over. Theres a number of ways to go about this.